base/reaction/pom.xml

```diff
@@ -91,6 +91,13 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>cdk-valencycheck</artifactId>
+            <version>${project.parent.version}</version>
+            <scope>test</scope>
+            <type>test-jar</type>
```

Comment:
I'm not sure this is cleaner... after reading through the rest the new "extends ValencyCheckTest" doesn't seem logical to me since it's in a lot of places which are unrelated to valency checking.